templates/pulpcore-api.service.erb

```diff
@@ -0,0 +1,35 @@
+[Unit]
+Description=Pulp WSGI Server
+After=network-online.target
+Wants=network-online.target
+
+[Service]
+Environment="DJANGO_SETTINGS_MODULE=pulpcore.app.settings"
+Environment="PULP_SETTINGS=/etc/pulp/settings.py"
+Environment="PATH=/usr/local/lib/pulp/bin:"
+User=pulp
+PIDFile=/run/pulpcore-api.pid
```

Comment:
I suspect this is a copy of what the ansible installer has and that's fine but https://docs.gunicorn.org/en/stable/deploy.html#systemd suggests deploying on a unix socket. It also suggests pid files aren't needed when using systemd. Probably best to keep this for the next iteration.